plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -300,15 +303,21 @@ def _process_manifest(self, manifest, digest, available_blobs, tag=None):
             manifest_file.write(manifest)
         manifest = models.Manifest.from_json(manifest, digest)
         self.parent.available_manifests.append(manifest)
+        foreign_layer = 'application/vnd.docker.image.rootfs.foreign.diff.tar.gzip'
```

Comment:
This would be good to call out as a constant, either in a constants module, or at least at the top of this file.